Av1an/compose.py

```diff
@@ -9,7 +9,7 @@
 from .vvc import get_yuv_file_path
 
 
-def gen_pass_commands(args: Args, chunk: Chunk) -> List[str]:
+def gen_pass_commands(args: Args, chunk: Chunk) -> List[List]:
```

Comment:
This return type is wrong.
```suggestion
def gen_pass_commands(args: Args, chunk: Chunk) -> Tuple[Tuple[str]]:
```